src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable consistent-return */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `eslint-disable` is generally discouraged as it can lead to ignoring important warnings or errors that ESLint would otherwise catch. If there's a specific rule you want to disable, consider using `eslint-disable-next-line [rule]` or `eslint-disable-line [rule]` instead. This way, you only disable the specific rule for the specific line where it's needed.

```diff
- /* eslint-disable consistent-return */
+ // appropriate eslint-disable-next-line or eslint-disable-line usage
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->